test/path.jl

```diff
@@ -89,22 +89,21 @@
         @test ["a", "b", "c"] == splitpath(joinpath("a","b","c"))
         @test [] == splitpath("")
 
+        # Unix-style paths are understood by all systems.
+        @test ["/", "a", "b"] == splitpath("/a/b")
+        @test ["/"] == splitpath("/")
+        @test ["a"] == splitpath("a/")
+        @test ["a", "b"] == splitpath("a/b/")
+        @test ["/"] == splitpath("///")
+        @test ["/", "a", "b"] == splitpath("///a///b///")
+
         if Sys.iswindows()
-            @test ["C:", "a", "b", "c"] == splitpath("C:\\\\a\\b\\c")
-            @test ["C:"] == splitpath("C:\\\\")
-            @test ["J:"] == splitpath("J:\\\\")
+            @test ["C:\\", "a", "b", "c"] == splitpath("C:\\\\a\\b\\c")
+            @test ["C:\\"] == splitpath("C:\\\\")
+            @test ["J:\\"] == splitpath("J:\\\\")
```

Comment:
Add a test for a relative path with a drive? (Yes, this is a thing, weirdly enough.)

Reply:
What do you mean by this? Can you give me an example?

Reply:
`C:`, `C:a`, `C:a\b`

Reply:
Done, thanks!